experimental/grouped-and-auto-accessors.md

```diff
@@ -0,0 +1,85 @@
+# [Grouped Accessors and Auto-Accessors][proposal-grouped-and-auto-accessors]
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition | StaticBlock | ClassAccessor ];
+}
+```
+
+## ObjectExpression
+
+```js
+extend interface ObjectExpression {
+    properties: [ Property | ObjectAccessor ];
+}
+```
+
+## ClassAccessor
+
+```js
+interface ClassAccessor <: PropertyDefinition {
+    type: "ClassAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
```

Comment:
`ClassAccessor` should also have a `value` field which is similar to a class field's value/initializer:

```js
accessor foo { get; set; } = 123;
```

Reply:
Ah, I see, it inherits these properties. I think this makes sense.